discovery_engine_core/web-api/openapi.yaml

```diff
@@ -177,6 +177,7 @@ components:
           properties:
             kind:
               type: string
+              enum: [not_enough_interactions]
```

Comment:
:pushpin: Slightly off topic by why did we move to naming thinks in snake case which make to rust types, bother here and in the db. Shouldn't it be `NotEnoughInteractions`?

Reply:
i don't know, i just followed the naming style of the already existing enums of the spec. if we should change the name style, i guess it makes sense to have a follow up ticket for that.

Reply:
So I guess we accidentally did change the naming style at some point? :sweat_smile:  

I just wanted to ask, because in the actix PR I added a small macro which basically "derives" the error type and it uses `stringify!($name)` and converting that to snake case requires using a proc-macro or unnecessary runtime conversions.